portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -54,6 +54,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // Get the input from the form.
+    //Translate
+    String lang = request.getParameter("language");
+    System.out.println(lang);
```

Comment:
I think this should technically go in doGet(). If I understand your feature correctly, when a user posts a comment, your server should store it as is, in its original form. Translation occurs when you get the comments; when responding to a GET request, the server should check "oh, is there a specific language the client wants these comments in?" and return the translated comments then (as opposed to translating the comments before you're about to store them?).